gradle.properties

```diff
@@ -1,2 +1,2 @@
-pravegaCommit=4bf247f
-pravegaVersion=0.7.0-rc0
+pravegaCommit=355579c
+pravegaVersion=0.8.0-pre-release
```

Comment:
I think we should change that configuration with respect to Pravega 0.8.0 release information: https://github.com/pravega/pravega/releases/tag/v0.8.0
```
pravegaCommit=e4c436b
pravegaVersion=v0.8.0
```

Reply:
fair enough